README.md

```diff
@@ -127,6 +132,8 @@ target is a url (http load tests) or host:port (grpc health test).  flags are:
 	List of pXX to calculate (default "50,75,90,99,99.9")
   -payload string
 	Payload string to send along
+  -ping
```

Comment:
I thought the `load` command with the `-grpc` flag is responsible for performing a grpc load test?

Reply:
ack